src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -4019,7 +4019,14 @@ private ushort getRelocTypeHint(void* target)
                     return (ushort)RelocType.IMAGE_REL_BASED_REL32;
 
                 case TargetArchitecture.ARM:
+#if READYTORUN
                     return (ushort)RelocType.IMAGE_REL_BASED_THUMB_BRANCH24;
+#else
+                    var targetObject = HandleToObject(target);
+                    if (targetObject is ExternSymbolNode)
```

Comment:
Note: I would love to use something like `targetObject is ExternMethodSymbolNode` or `targetObject is ExternSymbolNode and not RuntimeImportMethodNode`. Unfortunately, I cannot, because multiple C library calls are imported as `InternalCall`. That's the case at least for `memmove` and the float/double math functions like `ceil`.